lms/templates/wiki/edit.html

```diff
@@ -43,7 +43,7 @@
       </a>
 
   </div>
-  <section id="previewModal" class="modal" aria-hidden="true">
+  <div id="previewModal" class="modal" aria-hidden="true">
```

Comment:
These section tags don't really make sense. These are just hidden HTML that are shown as "modals" when a particular action is triggered. There are 4 of these in total.

Leaving them as "section" would require making a heading tag as the next element.